src/js/components/covid19/recipient/map/RecipientMapTooltip.jsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * ExplorerTooltip.jsx -> RecipientMapTooltip.jsx
+ * Created by Jonathan Hill 07/21/20
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { truncate } from 'lodash';
+import * as MoneyFormatter from 'helpers/moneyFormatter';
+
+const propTypes = {
+    y: PropTypes.number,
+    x: PropTypes.number,
+    name: PropTypes.string,
+    amount: PropTypes.object,
+    awards: PropTypes.object
+};
+
+export default class RecipientMapTooltip extends React.Component {
+    componentDidMount() {
+        this.positionTooltip();
+    }
+    componentDidUpdate() {
+        this.positionTooltip();
+    }
+
+    positionTooltip() {
```

Comment:
Wow is it really this easy to make a functioning tooltip? This is also a pretty complex use case. This makes me dislike our tooltipwrapper even more.

Reply:
This entire component is 100 lines. The `TooltipWrapper` is almost 300.